lib/ansible/modules/system/mount.py

```diff
@@ -9,9 +9,11 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-ANSIBLE_METADATA = {'metadata_version': '1.1',
-                    'status': ['preview'],
-                    'supported_by': 'core'}
+ANSIBLE_METADATA = {
```

Comment:
Please revert the non-functional changes in this PR, it makes reviewing the git blame very difficult when we are trying to identify bugs

Reply:
Thanks for your commend, John. Unfortunately even if I create a separate PR for the non-functional changes, the Git blame will change anyway. I recommend you to switch to the _Split view_ to be able to see changes better for this review.